### ISSUE TITLE
Config parsing errors using example for auth_notify_cmd and auth_error_cmd

### ISSUE THREAD

**Issue by simonm** [ltratt/pizauth#17]
Hi

I'm getting errors when starting the server when I include the example auth_notify_cmd and auth_error_cmd. Even reducing the complexity of the cmds to remove possible delimiting issues gives the same result.

Pizauth version. Compiles from source cloned from main, today (18th August):

```# pizauth info
pizauth version 1.0.1:
  cache directory: /home/<user>/.cache/pizauth
  config file: /home/<user>/.config/pizauth.conf
```

```
cat $HOME/.config/pizauth.conf: 

account "workbackup" {
    auth_uri = "https://login.microsoftonline.com/common/oauth2/v2.0/authorize";
    token_uri = "https://login.microsoftonline.com/common/oauth2/v2.0/token";
    client_id = "XXXXXXXXXXXXXXXXX";
    client_secret = "XXXXXXXXXXXXXXXXX";
    redirect_uri = "http://localhost/";
    scopes = [
      "https://outlook.office365.com/IMAP.AccessAsUser.All",
      "https://outlook.office365.com/SMTP.Send",
      "offline_access"
    ];
    auth_uri_fields = { "login_hint": "XXXX@XXXXXXXXXX" };
    auth_notify_cmd = "true";
    auth_error_cmd = "true";
}
```

Error on running `pizauth server` with the above config:

```
# pizauth server
Parsing error at line 13 column 5. Repair sequences found:
   1: Insert }
Parsing error at line 15 column 1. Repair sequences found:
   1: Delete }
```

Is there something obvious I'm doing wrong? Removing those last two config lines resolves the issue.

**Comment by simonm** [comment]
That makes sense. I will look again. Thank you

**Comment by simonm** [comment]
If I put the example config for `auth_*_cmd` outside of the workaccount block then the config is parsed fine.
However, I do not see that notify-send is actually sending anything.
With `pizauth server -dv` I get the following via STDOUT:

```
ERROR - workbackup: error when running 'notify-send -t 30000 'pizauth authentication' "<a href=\"${echo $PIZAUTH_URL | sed 's/&/&amp;/g'}\">$PIZAUTH_ACCOUNT</a>"':
```

I'm sure this is a result of me not understanding the way the config works...

**Comment by ltratt** [comment]
> the ${ .. } should be $( .. ) in the auth_notify_cmd

Oops -- you're quite right! Embarrassingly my local `pizauth.conf` uses `$(...)` but I didn't fix the website or the README. The website is fixed and the README is now fixed in 9c2ed02 -- thanks for persevering with this!
